.gitignore

```diff
@@ -15,6 +15,7 @@ _test
 # OS or Editor folders
 .komodotools
 *.sublime-*
+.vscode/
 
```

Comment:
some maintainers don't like this, I personally don't mind it a bit. Thanks for caring enough to do this.